ui/src/components/core/input/input-file.tsx

```diff
@@ -0,0 +1,94 @@
+import { Flex } from '../../layout';
+import { dripStitches } from '../../../theme';
+import { forwardRef, useRef, useState } from 'react';
+import { Icon } from '../icon';
+import { Form } from '../../../components/form/form';
+
+const { styled } = dripStitches;
+
+const BorderInput = styled('div', {

```

Comment:
Shouldn't these come from another file?

Reply:
Not necessarily since it is being used just inside of this component.